lib/formatters/tapFormatter.mjs

```diff
@@ -17,6 +18,7 @@ export default function tapFormatter(results) {
 		const ignored = result.ignored ? ' # SKIP ignored' : '';
 		const error = result.errored ? 'not ' : '';
 
+		result.source = getEscapedSource(result.source);
 		lines.push(`${error}ok ${index + 1} - ${result.source}${ignored}`);
```

Comment:
```suggestion
		lines.push(`${error}ok ${index + 1} - ${getEscapedSource(result.source)}${ignored}`);
```

Let's avoid this mutation.
It can only be a source of bugs and confusion.
It can never be a good approach.

Anything that needs to use the value of `result.source` beyond this point will get a TAP escaped value.

That is only useful for TAP reporters, such as this formatter.

Anything else must get the original value.